modules/mtc-deploying-a-vm.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+// * migration_toolkit_for_containers/mtc-migrating-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="mtc-deploying-a-vm_{context}"]
+= Deploying a Virtual Machine
+
+When you have installed {VirtProductName} and CDI, and they are active, create a namespace.
+
+In this example, we use the namespace `mig-vm`.
+
+.Procedure
+
+* Deploy the following YAML to create a Fedora virtual machine (VM).
+
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: rhel9-lime-damselfly-72
+  namespace: default
```

Comment:
we need to remove `namespace: default`, migration is not working in default namespace

Reply:
In this example, we use the namespace mig-vm.  -- then we could update to `namespace: mig-vm`